src/reff.js

```diff
@@ -64,14 +64,23 @@ export function reff(output, rt, beta, population, mixingMatrix) {
     throw Error("mismatch between population and mixing matrix size");
   }
 
-  const tNow = rt.length;
+  if (beta.length !== rt.length) {
+    throw Error("mismatch between rt and beta size");
+  }
+
+  if (tSubset == null) {
+    tSubset = [...Array(rt.length).keys()];
```

Comment:
this is so weird :)

Reply:
Isn't it bonkers that JS doesn't have seq?